docker-compose.yaml

```diff
@@ -10,6 +10,7 @@ services:
     image: ankerctl/ankerctl:latest
     container_name: ankerctl
     restart: unless-stopped
+    pull_policy: build
```

Comment:
Allows to always pull the Git repo if you are building an image from a repo instead of local files. Example: I use Portainer to manage my images, I give it a URL of a Git Repo and tell it where in the repo to find the Dockerfile and it pulls the repo and builds. Having a `pull_policy` tell it to always pull the repo on build.

Reply:
that's a pretty specific use case. I suspect most people are going to pull the docker container from whatever repo we end up placing it in.